src/init.cpp

```diff
@@ -108,22 +115,45 @@ void Init<Backend::GPU>::finalize() {
 }
 #endif
 
-template <typename T>
-void updateConfigurationValue(hpx::program_options::variables_map const& vm, T& var,
-                              std::string const& env_var, std::string const& cmdline_option);
+template <>
+void Init<Backend::MC>::initialize(configuration const& cfg) {
+  // TODO: Consider disabling polling in finalize()
+  if (cfg.mpi_mech == comm::MPIMech::Polling) {
+    hpx::mpi::experimental::init(false, cfg.mpi_pool);
```

Comment:
I suppose this should be enabled independently of the backend.

Reply:
We don't need it for `MPIMech::Yielding`. Why would you prefer having it enabled unconditionally?

Reply:
Ugh, sorry. Ignore my comment. My mind was telling me that it won't be enabled for the CUDA case, but of course it will since the `MC` backend is always enabled.